### ISSUE TITLE
could you provide a test version?

### ISSUE THREAD

**Issue by zhangqizky** [vinthony/image-splicing-localization#3]
hi, vinthony~I am very interested in your work and thank u for opening it, I wonder that if you could give a script for testing your method with a pretrained model? if you could, it will be great. Thank u again in advance.

**Comment by vinthony** [comment]
@zhangqizky thanks for your attention. However, I lost the previous data and model because I graduate from  the master degree. Which dataset are you want to compare and I will try to rerun a pre-trained model recently.

**Comment by RightXu** [comment]
hi,vinthony~Recently,I am studying your code and thank u for opening it.I have some doubts and would like to ask you.If you could explain  the input parameters(center,scale,res) about the final_preds function in file evaluation.py? And in your paper, there is a loss of smoothness.However, I did not find the corresponding function in the code.Looking forward to your reply and thank you in advance !

**Comment by NeerajPise** [comment]
Could you provide a model or a method to create one that can be used to test on custom data?

**Comment by 111zzy** [comment]
hi, vinthony~I am very interested in your work and thank u for opening it, I want to compare the result on NC2016, but i conn't run the code. There some problem about the size of image. Could you provide a  pre-trained model? If you could, it will be great. Thank u again in advance.
